File: Level_D/d050.cs
using System;

public class Pz{
    public static void Main(){
        var arr = Console.ReadLine().Trim().Split(' ');
        int d1 = int.Parse(arr[0]);
        int d2 = int.Parse(arr[1]);
        d1 = (d1 > 5) ? 5 : d1;
        d2 = (d2 > 5) ? 5 : d2;
        Console.WriteLine(d1 + d2);
    }
}

File: Level_D/Python2/d050.py
# coding: utf-8

if __name__=="__main__":
    arr = raw_input().strip().split(' ')
    n = 0
    for s in arr:
        n1 = int(s)
        if n1 > 5:
            n += 5
        else:
            n += n1

    print(n)
